bazel/v8.patch

```diff
@@ -79,17 +79,30 @@ index ce3f569fd5..dc8a4c4f6a 100644
 --- a/src/wasm/c-api.cc
 +++ b/src/wasm/c-api.cc
 @@ -2238,6 +2238,8 @@ auto Instance::exports() const -> ownvec<Extern> {
- 
+
  }  // namespace wasm
- 
+
```

Comment:
Why is C++ code influenced by Python version?

Reply:
my editor removed whitespace - i can put it back. This is one of those reasons that glint should be legally enforced, patching repos with messy whitespace issues is a nightmare